### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kaywilliams**
Modified the motivation section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
 At the same time, it is a single place where one can find with ease, which COSE library
 is best suited for ones project specific needs.
 
-# Need for Gluecose?
-COSE is scary. COSE has a very wide feature spectrum. It is quite complex.
-Complexity multiplied with a vast range of features amplify the trepidation.
-Hence when a software developer wants to pick a COSE Library for their project,
-one does not know, what one gets ?
+# Motivation
+COSE has a very wide feature spectrum and is quite complex. This makes it difficult for library implementors to test and verify their libraries. Also, many implementors choose to implement a subset of the full COSE specification. This makes it difficult for developers to know which COSE library to select given their project's needs.
```
Comment (henkbirkholz) [inline README.md]: So what is the motivation to use COSE? :-) I mean, it obvious to the people building global resilient infrastructure, but not necessarily to the interested first time reader. (that's why we are doing gluecose amongst other things)

File: README.md [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
+The COSE signing envelope format, which is broadly applicable from small devices to large-scale server environments, attempts to keep the flavor of [Javascript Object Signing and Encryption](https://datatracker.ietf.org/wg/jose/documents/) (JOSE) specifications ([JWS](https://datatracker.ietf.org/doc/rfc7515/), [JWE](https://datatracker.ietf.org/doc/rfc7516/), [JWK](https://datatracker.ietf.org/doc/rfc7517/), [JWA](https://datatracker.ietf.org/doc/rfc7518/)) while providing additional features and flexibility. To meet requirements for device reach and flexibility, COSE has a very wide feature set and is quite complex. This makes it difficult for library implementors to test and verify their libraries. Also, many implementors choose to implement a subset of the full COSE specification. This makes it difficult for developers to know which COSE library to select given their project's needs.
```
Comment (yogeshbdeshpande) [inline README.md]: To meet requirements for device reach and flexibility,

This line can be slightly better re-worded as:
To meet the requirements for a broad range of use cases (Devices, Edge or Servers)  and flexibility, COSE offers a wide feature set and is quite complex.  This makes it difficult for library implementers to test and verify their libraries. Also, many implementers choose to implement a subset of the full COSE specification.
This makes it difficult for application developers to know which COSE library to select given their project needs.

File: README.md (hunk 1 above)
Comment (kaywilliams) [reply README.md]: Added

File: README.md (hunk 2 above)
Comment (kaywilliams) [reply README.md]: Incorporated. Thanks Yogesh.

**Pull request merged**